pkg/jobs/jobspb/jobs.proto

```diff
@@ -663,6 +663,14 @@ message MigrationProgress {
 
 }
 
+message SQLStatsCompactionDetails {
+  bool disable_follower_read = 1;
```

Comment:
Do we really need this in the job details? From my reading it only comes from a testing knob which should be shared by all of the instances of the job.